src/effects/VST/VSTEffect.cpp

```diff
@@ -3463,4 +3463,94 @@ void VSTEffectWrapper::ForEachParameter(ParameterVisitor visitor) const
 }
 
 
+bool VSTEffectWrapper::FetchSettings(VSTEffectSettings& vst3settings) const
```

Comment:
Comment in the commit that `FetchSettings` adapts `SaveSettings` and `SaveParameters` but storing in-memory in EffectSettings, not in a config file.

Reply:
ok, done